_includes/help-build-fail.md

```diff
@@ -6,4 +6,29 @@ The files we are using are very picky. If you made any mistakes, you will see a
 
 When you commit your changes, Travis CI will re-check your file.
 
-Keep doing this until you see a message that the build has passed. If you need help, don't forget you can find us in chat or @ mention githubteacher in your pull request.
+Keep doing this until you see a message that the build has passed. If you need help, don't forget you can find us in chat or @ mention `@githubteacher` in your pull request.
+
+### Help With Specific Issues
+
+The build can fail because of many reasons. By looking at the details of the **Travis CI** log it might help you to troubleshoot your code.
+
+1. Open the _Pull request_  
+2. Go to the bottom to the "merge" icon to the section where it says that 'All checks have failed'.  
+  Here you'll see a lot of red crosses. Not good.  
+3. Open the `Details`  
+
+The build log of Travis CI may appear a little cryptic. But with some careful reading you might be able to pull out the reason why your build is failing. (Btw if you think the `job log` is cryptic, try and click on the `Raw log`.)
```

Comment:
Could we change "Btw" to "by the way"? Since not all users are native english speakers, some abbreviations may not be well understood.